client/apps/edit/components/section_image_set/index.coffee

```diff
@@ -94,16 +94,18 @@ module.exports = React.createClass
     e.preventDefault()
     val = @refs.byUrl.value
     slug = _.last(val.split '/')
-    @refs.byUrl.setState loading: true
+    $(@refs.byUrl).siblings('button').addClass('is-loading')
```

Comment:
Using jQuery to manipulate the DOM like this is typically an anti-pattern in React-land and usually you'd want to reflect UI updates by branching logic within the `render` function. This style of jQuery code is referred to as "imperative" programming whereas React encourages "declarative" programming—that is instead of saying "find this element, change this class" React would encourage saying "the UI looks like this when the state is updated". That said, I know this is a quick time-sensitive fix so I'm not going to hold it up—but we should dig in more when we pair tomorrow.